drand/drand.go

```diff
@@ -122,9 +122,11 @@ func New(host p2p.Host, ShardID uint32, peers []p2p.Peer, leader p2p.Peer, confi
 	}
 
 	// VRF keys
-	priKey, pubKey := p256.GenerateKey()
-	dRand.vrfPriKey = &priKey
-	dRand.vrfPubKey = &pubKey
+	sk := vrf_bls.NewVRFSigner(blsPriKey)
+	pk := vrf_bls.NewVRFVerifier(blsPriKey.GetPublicKey())
+	dRand.vrfPriKey = &sk
```

Comment:
We should instead reuse the blsKey in the consensus object. Shouldn't generate a separate bls key anymore.

Reply:
The node consensus private key (blsPrivKey) is reused here. It's just been typecast-ed to the base rypto/vrf interface. 

func NewVRFSigner(seck *bls.SecretKey) vrf.PrivateKey {
	return &PrivateKey{seck}
}